html/modules/custom/reliefweb_job_tagger/reliefweb_job_tagger.permissions.yml

```diff
@@ -5,3 +5,7 @@ enforce ocha ai job tag:
 bypass ocha ai job tag:
   title: 'Bypass OCHA AI Job tag'
   description: 'Allow users to ignore Access OCHA AI Chat advanced features.'
+
+test ocha ai job tag:
```

Comment:
Should this be assigned to the "Editor" role [in the config](https://github.com/UN-OCHA/rwint9-site/blob/develop/config/user.role.editor.yml)?

Reply:
I was planning to do it manually